---
title: Cryptic Species Hide in Plain Sight
layout: page
author: hbroadley
permalink: /2016-06-06-cryptic-species-hide-in-plain-sight-hbroadley/
tags:
- dna
- genetics
- biodiversity
- species
categories:
 - Earth's Organisms
header:
 image_fullwidth: broadley_06062016_fig1.jpg
image:
 thumb: broadley_06062016_fig2.jpg
teaser: The advent of affordable DNA sequencing has put molecular techniques at the forefront of species discovery.  For the sake of biodiversity, ecological conservation, public health, and pest management, let’s hope that “new age of [species] discovery” has only just begun.
subheadline: Some species hide in plain sight, but scientists have ways to suss them out
source-id: 1kLZsj_K_XKa-2EzKMANFh3hDeDldqXoCAUimNsqh7rM
published: true
---
**_Discovering New Species_**

In the last decade, there has been a lot of talk about the 'new age of (species) discovery.' While biodiversity is threatened by habitat loss, pollution, and climate change, amazingly we are discovering and naming new species at a faster rate than any other time since the mid-1700s [1]. This includes the discovery of the kipunji, a new monkey in southwestern Tanzania [2], *A. johnnycashi*, the newly classified southwestern U.S. tarantula named after Johnny Cash [3], and a completely new family of passerine birds (perching or songbirds) represented by the charmingly small spotted wren-babbler from China [4] (Fig. 1). 

![image alt text]({{ site.url }}/public/9dDIm9M6PjD8uff9f8HxSQ_img_0.jpg)
**Fig. 1**  The (a) Kipunji, a newly discovered Old World monkey, (b) *A. johnnycashi* tarantula female, and (c)  the spotted wren-babbler.  *Image credits:* a and c: Wikipedia commons, b: Hamilton, Hendrixson, and Bond 

One might expect that now, after 250 years of taxonomic efforts and modernization, we would have discovered all such charismatic species, but a number of factors are driving the current rate of new discoveries.  First, we now have access to regions that were out of reach before.  Improved international relations and the construction of roads (development) have provided scientists with greater access to study in remote habitats. Helicopters, drones, satellite technologies, and deep-sea dives have all opened up previously unreachable territories.  Secondly, there is a palpable sense of urgency now to discover, document, and understand our biodiversity before it is lost to us.  Recent extinction rates are 100 to 1000 times greater than their pre-human levels [5]. Lastly, the advent of affordable DNA sequencing has put molecular techniques at the forefront of species discovery.  The Johnny Cash tarantula is an example of these techniques put to work.  Researchers used DNA analysis, along with information on geographic location, morphology, and behavior, from 3000 tarantulas collected across the Southwest United States to figure out where this species of tarantula fit within the family tree of spiders.  The researchers consolidated some groups, but also denoted 15 new species.  What had been declared a "nomenclatural and taxonomic nightmare"[6] has, with the aid of molecular tools, been resolved.  

**_DNA Barcoding_**

Molecular techniques utilize the unique genetic code found in an individual organism's DNA.  The amount of difference in an individual's genetic code from other individuals reflects its relatedness to those other individuals. For a given section of DNA, the sequences from individuals within the same species are more similar than sequences across different species.  Loosely speaking, the more similar the DNA sequences to each other, the more closely related those individuals are to each other evolutionarily.  Similarly, more differences in the genetic code between two individuals reflect larger evolutionary isolation from one another.  For example, in humans there is only about 0.1% difference in DNA code between individuals, but there is about 1.2% different between human DNA sequences and that of chimpanzees.  There is an 8% difference between the DNA of a human and a mouse and 50% difference between humans and the fruit fly. 

 

An important advance in the field of molecular taxonomy is the development of DNA barcoding. This approach characterizes a small segment of an organism's DNA to identify it as belonging to a particular species and to connect it to other closely related species.  The region of DNA typically examined for barcoding is a mitochondrial gene called cytochrome c oxidase subunit 1.  That is a mouthful so it is simply referred to as the COI region. This region of DNA was selected because it provides large variation between species, but relatively small variation within a species. To visualize the DNA, a series of steps are undertaken: the DNA region of interest is targeted, amplified, and sequenced so as to be able to visualize the DNA code and compare it to other codes to compare and contrast similarities and differences (Fig. 2). Using this approach, large numbers of collected organisms can be tested relatively rapidly.  Additionally, museum collections can be tested as was done for the tarantula survey [3].  The DNA barcoding technique offers the possibility of quick identification, phylogenetic clarification, and the discovery of new species.  

By peering into genetic code, again and again this technique has shown that what scientists had thought to be a single widely distributed species is not one species at all, but rather a collection of small populations that look alike but are genetically and evolutionarily distinct (see *S. fulgerator* CELT and TRIGO in Fig. 2).  Because they look outwardly similar to another species they are called *cryptic species*. The cryptic species are hidden within what is referred to as a species complex.  In most cases the species complex is made up of species that share a common ancestor but have diverged slightly over the course of evolution. 

![image alt text]({{ site.url }}/public/9dDIm9M6PjD8uff9f8HxSQ_img_1.png)
**Fig. 2**  DNA Barcodes show differences and similarities within and between groups to help delineate species and uncover hidden diversity.  *Image credit*: [IDT "Barcoding Life"](https://www.idtdna.com/pages/decoded/decoded-articles/your-research/decoded/2012/06/15/barcoding-life)   

**_Why it matters_**

Detecting cryptic species is important for a number of reasons.  The existence of cryptic species is hiding species diversity, resulting in underestimates of local and global biodiversity.  We may be far more ignorant of our planet's rich species diversity than we had originally thought. Discovering cryptic species is essential for conservation ecology. Additionally, it is essential that we can recognize similar but distinct species for disease and pest control. To combat emerging diseases, scientists need to be able to detect and correctly identify the acting pathogens.  Similarly, in the insect world DNA is a powerful tool for detecting potentially destructive invasive insect introductions or pest insect populations resulting from human-altered selection pressures. For example, specific subpopulations of greenbug aphid populations have been found to specialize on particular crops [7]. Knowledge such as this is essential for effective insect pest management.   

For the sake of biodiversity, ecological conservation, public health, and pest management, let's hope that "new age of [species] discovery" has only just begun. 

**References**

1. Donoghue, M. J., and W. S. Alverson. 2000. A New Age of Discovery. *Annals of the Missouri Botanical Garden* 87(1): 110-26. [http://donoghuelab.yale.edu/new-age-discovery](http://donoghuelab.yale.edu/new-age-discovery) 

2. Jones, T., C. L. Ehardt, T. M. Butynski, T. R. B. Davenport, N. E. Mpunga, S. J. Machaga, and D. W. De Luca. 2005. The Highland Mangabey Lophocebus Kipunji: A New Species of African Monkey. *Science* 308(5725): 1161-64. [http://www.ncbi.nlm.nih.gov/pubmed/15905399](http://www.ncbi.nlm.nih.gov/pubmed/15905399) 

3. Hamilton, Chris A., Brent E. Hendrixson, and Jason E. Bond. 2016. Taxonomic Revision of the Tarantula Genus Aphonopelma Pocock, 1901 (Araneae, Mygalomorphae, Theraphosidae) within the United States. *ZooKeys* 560: 1-340. [http://zookeys.pensoft.net/articles.php?id=6264](http://zookeys.pensoft.net/articles.php?id=6264) 

4. Alstrom, P., D. M. Hooper, Y. Liu, U. Olsson, D. Mohan, M. Gelang, H. L. Manh*, et al.* 2014. Discovery of a Relict Lineage and Monotypic Family of Passerine Birds. *Biology Letters* 10 (3)  [http://rsbl.royalsocietypublishing.org/content/10/3/20131067](http://rsbl.royalsocietypublishing.org/content/10/3/20131067) 

5. Pimm, S. L., G. J. Russell, J. L. Gittleman, and T. M. Brooks. 1995. The Future of Biodiversity. *Science* 269(5222): 347-50. [http://science.sciencemag.org/content/269/5222/347](http://science.sciencemag.org/content/269/5222/347) 

6. Raven, R. J. 1990. Comments on the Proposed Precedence of Aphonopelma Pocock, 1901 (Arachnida, Araneae) over Rhechostica Simon, 1892. *Bulletin of Zoological Nomenclature* 47(2): 126-27.

7. Anstead, J. A., J. D. Burd, and K. A. Shufran. 2002. Mitochondrial DNA Sequence Divergence among Schizaphis Graminum (Hemiptera : Aphididae) Clones from Cultivated and Non-Cultivated Hosts: Haplotype and Host Associations. *Bulletin of Entomological Research* 92(1): 17-24. [http://www.ncbi.nlm.nih.gov/pubmed/12020358](http://www.ncbi.nlm.nih.gov/pubmed/12020358) 

## More From Thats Life [Science]
{: .t60 }
{% include list-posts offset="1" category="Earth's Organisms" %}
